public/index.ts

```diff
@@ -88,8 +86,7 @@ const TEST_DATA: Record<string, TestDataSpec> = {
 
 let view3D: View3d;
 
-const volumeCache = new VolumeCache(CACHE_MAX_SIZE);
-const requestQueue = new SubscribableRequestQueue(CONCURRENCY_LIMIT, PREFETCH_CONCURRENCY_LIMIT);
+const loadWorker = new LoadWorker(CACHE_MAX_SIZE, CONCURRENCY_LIMIT, PREFETCH_CONCURRENCY_LIMIT);
```

Comment:
this feels a bit funky to me.  LoadWorker looks like something that should not be exposed to outside clients - it should get an API name that sounds like some important and official part of the volume-viewer,  like VolumeLoaderSubsystem or VolumeLoaderFactory?  I'll read further and digest some more and maybe come back and edit this comment...

Reply:
Coming back to this, it does seem like VolumeLoaderFactory is a good name, since the main point of the class is to call `createLoader`.  We should document it as one of the key top level exports of the volume-viewer library, like Volume, View3D etc.

Reply:
Peyton and I discussed this a bit and my favorite idea was `VolumeLoaderContext`. I don't 100% agree that "loader" is too behind-the-scenes a term to be included in the public API, but I definitely think this PR could use some work to disambiguate all the items named with the words "loader," "worker," "volume" in some combination.